### ISSUE TITLE
H2 should be H1 on manage reading groups page (and other pages in this section)

### ISSUE THREAD

**Issue by zdavis** [ManifoldScholar/manifold#2394]
![image](https://user-images.githubusercontent.com/255909/66520894-e210e080-ea9e-11e9-8692-9e9b2c8256fc.png)
